Hashing/RomanNumberToInteger

```diff
@@ -0,0 +1,20 @@
+class Solution:
+    # @param A : string
+    # @return an integer
+    def romanToInt(self, A):
+        if A=="":
+            return 0
+        dictionary = {"I": 1, "V": 5, "X": 10, "L": 50, "C": 100, "D": 500, "M": 1000}
+        totalSum = 0
+        firstChar = dictonary[A[-1]]
```

Comment:
@sank8dalvi 
There's a typo. Please take your time and verify it properly. :)

Reply:
Very sorry for this issue and I have fixed it.